dashboard/app/dsl/script_dsl.rb

```diff
@@ -88,6 +92,8 @@ def parse_output
       project_widget_visible: @project_widget_visible,
       project_widget_types: @project_widget_types,
       script_announcements: @script_announcements,
+      new_name: @new_name,
+      family_name: @family_name,
```

Comment:
I expected this to come with updated tests in [dsl/dsl_test.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/dsl/dsl_test.rb).

Reply:
I will add some tests for this.